example/sns.tf

```diff
@@ -34,3 +38,17 @@ resource "kubernetes_secret" "example_sns_topic" {
   }
 }
 
+# Example for pushing additional user/access key secrets into namespaces 
+#
+# resource "kubernetes_secret" "team_1_secret" {
```

Comment:
Is it worth using a `for_each` here too, so you don't have to create a new secret for each team?